src/main/java/io/sdkman/broker/download/CandidateDownloadHandler.java

```diff
@@ -47,12 +58,44 @@ public void handle(Context ctx) {
                                             downloadResolver.resolve(downloads, p.name())
                                                     .ifPresentOrElse(v -> {
                                                         audit(details, p.id(), v.platform());
+                                                        writeChecksumHeaders(ctx, v.checksums());
                                                         ctx.redirect(302, v.url());
                                                     }, clientError(ctx, 404))),
                             clientError(ctx, 400));
         }, clientError(ctx, 404));
     }
 
+    private void writeChecksumHeaders(Context ctx,
```

Comment:
The above change will simplify the method signature to something like:
```suggestion
    private void writeChecksumHeaders(Context ctx,
        scala.collection.immutable.Map<String, String> checksums) {
```